packages/use-i18n/src/__tests__/usei18n.tsx

```diff
@@ -250,6 +250,89 @@ describe('i18n hook', () => {
     })
   })
 
+  describe('getCurrentLocale', () => {
+    it('should set current locale from localStorage', async () => {
+      jest.spyOn(global, 'navigator', 'get').mockReturnValueOnce({
+        languages: ['fr'],
+      } as unknown as Navigator)
+      const mockGetItem = jest.fn().mockImplementation(() => 'en')
+      const mockSetItem = jest.fn()
+      const localStorageMock = jest
+        .spyOn(global, 'localStorage', 'get')
+        .mockReturnValue({
+          getItem: mockGetItem,
+          setItem: mockSetItem,
+          clear: jest.fn(),
+        } as unknown as Storage)
+
+      const { result } = renderHook(() => useI18n(), {
+        wrapper: wrapper({
+          defaultLocale: 'es',
+          supportedLocales: ['en', 'fr', 'es'],
+        }),
+      })
+
+      await waitFor(() => {
+        expect(result.current.currentLocale).toEqual('en')
+      })
```

Comment:
Could we add `expect`s for the above localStorage mocks to make sure the proper method is called/not called?